test/qpy_compat/test_qpy.py

```diff
@@ -129,9 +128,11 @@ def generate_random_circuits():
 
 
 def generate_string_parameters():
-    """Generate a circuit from pauli tensor opflow."""
-    op_circuit = (X ^ Y ^ Z).to_circuit_op().to_circuit()
-    op_circuit.name = "X^Y^Z"
+    """Generate a circuit for the XYZ pauli string."""
+    op_circuit = QuantumCircuit(3, name="X^Y^Z")
+    op_circuit.z(0)
+    op_circuit.y(1)
+    op_circuit.x(2)
     return op_circuit
```

Comment:
I think this change is actually breaking the intended test; in the original, there's a `PauliGate` in the circuit, which has a `.params` field that contains a string.  The new form doesn't have that; it's just got a string in the `op_circuit` name, which is the standard.

We could probably hew to the intent of the test with something like
```python
from qiskit.circuit.library import PauliGate

op_circuit = QuantumCircuit(3)
op_circuit.append(PauliGate("XYZ"), op_circuit.qubits, [])
```

Reply:
Thanks. I use PauliGate 97f5a84.